tests/util/test_traceutils.py

```diff
@@ -103,17 +102,17 @@ def test_is_service_or_endpoint_ignored() -> None:
     agent.options.ignore_endpoints.append("service2.endpoint1")
 
     # ignore all endpoints of service1
-    assert is_service_or_endpoint_ignored("service1")
-    assert is_service_or_endpoint_ignored("service1", "endpoint1")
-    assert is_service_or_endpoint_ignored("service1", "endpoint2")
+    assert agent._HostAgent__is_service_or_endpoint_ignored("service1")
+    assert agent._HostAgent__is_service_or_endpoint_ignored("service1", "endpoint1")
+    assert agent._HostAgent__is_service_or_endpoint_ignored("service1", "endpoint2")
 
     # case-insensitive
-    assert is_service_or_endpoint_ignored("SERVICE1")
-    assert is_service_or_endpoint_ignored("service1", "ENDPOINT1")
+    assert agent._HostAgent__is_service_or_endpoint_ignored("SERVICE1")
+    assert agent._HostAgent__is_service_or_endpoint_ignored("service1", "ENDPOINT1")
 
     # ignore only endpoint1 of service2
-    assert is_service_or_endpoint_ignored("service2", "endpoint1")
-    assert not is_service_or_endpoint_ignored("service2", "endpoint2")
+    assert agent._HostAgent__is_service_or_endpoint_ignored("service2", "endpoint1")
+    assert not agent._HostAgent__is_service_or_endpoint_ignored("service2", "endpoint2")
 
     # don't ignore other services
-    assert not is_service_or_endpoint_ignored("service3")
+    assert not agent._HostAgent__is_service_or_endpoint_ignored("service3")
```

Comment:
If you removed the method `is_service_or_endpoint_ignored()` from the `src/instana/util/traceutils.py` file, this whole test is no longer valid. You can remove it.

Now, since you moved the method `is_service_or_endpoint_ignored()` to the `src/instana/agent/host.py`, you can move this whole function to the `tests/agent/test_host.py` file. BTW, I am missing a test for the method `filter_spans()` from the  `src/instana/agent/host.py`.